rttest/src/rttest.cpp

```diff
@@ -330,7 +330,7 @@ int Rttest::read_args(int argc, char ** argv)
             }
           }
 
-          long_to_timespec(nsec, &update_period);
+          uint64_to_timespec(nsec, &update_period);
```

Comment:
Here, nsec is set to a `size_t`, which isn't guaranteed to be a `uint64` (particularly on 32-bit).  I think the fix is easy enough; change the type of `nsec` above to `uint64_t`.  You may also have to change the `stol` to a `stoll` or even a `stoul`.

Reply:
Thank you for comment.
I decided to use `stoull` and push 80145260875df80b8efc90292b2c6f830142683e.